src/components/ColorTokenTable/color-token-table.scss

```diff
@@ -0,0 +1,128 @@
+.color-token-table {
+  display: flex;
+  flex-direction: column;
+}
+
+.color-token-table__theme-switcher {
+  margin-bottom: $spacing-xl;
+  justify-content: space-between;
+  background: $ibm-colors__white;
+
+  @include breakpoint('md') {
+    justify-content: flex-start;
+    background: transparent;
+  }
+}
+
+.color-token-table__theme-switcher
+  .bx--content-switcher-btn:not(.bx--content-switcher--selected) {
+  background-color: $ibm-colors__white;
+  color: $ibm-colors__gray--100;
+
+  &:hover {
+    background-color: $ibm-colors__gray--30;
+  }
+}
+
+.color-token-table__theme-switcher .bx--content-switcher-btn {
+  padding: $spacing-xs $spacing-xl $spacing-xs $spacing-md;
+  flex: 1;
+
+  @include breakpoint('sm') {
+    padding-right: $spacing-md;
+  }
+
+  @include breakpoint('md') {
+    padding-right: 4rem;
+    flex: none;
+  }
+}
+
+.color-token-value {
+  display: flex;
+  justify-content: space-between;
+}
+
+.color-token-value .bx--overflow-menu {
+  margin-left: $spacing-xs;
+}
+
+.color-token-value .bx--overflow-menu-options {
+  left: -9.25rem;
+  top: $spacing-xl;
+}
```

Comment:
If I don't add the CSS above for the `bx--overflow-menu-options` the menu floats to the right by 100px

Reply:
I just took a look at the overflow menu page and the styling is off there also. @asudoh any ideas on this? 
<img width="787" alt="screen shot 2019-01-10 at 2 21 44 pm" src="https://user-images.githubusercontent.com/2753488/50994941-345aac00-14e3-11e9-89ff-50e6feafe7a2.png">